yigitcanTask/force-app/main/default/classes/accountDetailsController.cls

```diff
@@ -0,0 +1,103 @@
+public class accountDetailsController {
+
+    public static final Integer ACCOUNT_NAME_MAX_LENGTH = 100;
+    public static final String ACCOUNT_LENGTH_ERROR_MESSAGE = 'Length is less than '+ACCOUNT_NAME_MAX_LENGTH+' characters';
+    
+    public List <Contact> contacts {get;set;}
+    public String searchKey {get;set;}
+    public Account account {get;set;}
+
+    public accountDetailsController(ApexPages.StandardController accountDetailsController ) {
+        this.account = (Account)accountDetailsController.getRecord();
+        this.contacts = this.getAllContacts();
+    }
+
+    public void primaryToContact() {
```

Comment:
Please rename method to smth that makes more sense
for example setPrimaryContact